traffic_ops/v4-client/cdn_notifications.go

```diff
@@ -0,0 +1,50 @@
+/*
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package client
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/apache/trafficcontrol/lib/go-tc"
+)
+
+const (
+	API_CDN_NOTIFICATIONS = apiBase + "/cdn_notifications"
+)
+
+// GetCDNNotificationsWithHdr returns a list of CDN Notifications.
+func (to *Session) GetCDNNotificationsWithHdr(cdnName string, header http.Header) ([]tc.CDNNotification, ReqInf, error) {
```

Comment:
This doesn't need to include `WithHdr` - there's no alternative without a header. You might also consider just allowing a `net/url.Values` instead of `cdnName`, so that the endpoint's full functionality is exposed, including pagination and sorting.